src/main/java/br/com/sbk/sbking/networking/server/Table.java

```diff
@@ -6,29 +6,34 @@
 import java.util.Collection;
 import java.util.HashMap;
 import java.util.Map;
-import java.util.concurrent.ExecutorService;
-import java.util.concurrent.Executors;
+import java.util.UUID;
 
 import br.com.sbk.sbking.core.Direction;
 import br.com.sbk.sbking.core.Player;
+import br.com.sbk.sbking.networking.server.gameServer.GameServer;
 
 public class Table {
 
-  private ExecutorService pool;
-  private static final int MAXIMUM_NUMBER_OF_PLAYERS_AND_KIBITZERS_IN_A_TABLE = 30;
-
   private Map<Direction, ClientGameSocket> playerSockets = new HashMap<Direction, ClientGameSocket>();
   private Collection<ClientGameSocket> spectatorSockets = new ArrayList<ClientGameSocket>();
+  private Map<UUID, ClientGameSocket> allSockets = new HashMap<UUID, ClientGameSocket>();
```

Comment:
Same comment on variable initialization outside constructors.

I have no personal preference on the matter, FWIW

Reply:
"Fixed" it here too.

I didn't find this as a [Checkstyle check](https://checkstyle.sourceforge.io/checks.html) so I will not add it as a rule. Anyway, will put them in constructors for legibility and maintainability from now on.